index.html

```diff
@@ -1466,6 +1462,13 @@ <h3>Types</h3>
 representation.
         </p>
 
+        <p class="note">
+The type system used in the data model described in this specification allows
+for multiple ways to associate types with data, implementers and authors are
```

Comment:
`data, implementers`
-> `data. Implementers`

Reply:
Fixed in 1b497225f4edd1ad81fe4c673099e4857b500b1b.